src/gpu/intel/conv/jit/zp_plan.cpp

```diff
@@ -134,14 +132,14 @@ class split_dispatcher_t {
                     && ends_with(c_layout, c_g_idx_, b)) {
                 simd_dim_idx_ = 1;
                 simd_ = b;
-                simd_tile_ = get_simd_tile(c_layout);
+                simd_tile_ = {{simd_dim_idx_, simd_}};
```

Comment:
Should we maybe keep `get_simd_tile` so that the code would look more uniform? See [L622](files#diff-04c5b9ad835d21302cdd10b58dba049daad15506dc24e73c49f0a71f8e6b6089R622).